src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -261,3 +274,32 @@ To use the EKCO add-on to remove a node:
    The EKCO purge script For information about the EKCO purge script, see [Purge Nodes](/docs/add-ons/ekco#purge-nodes) in _EKCO Add-on_.
 
 1. Remove the node from the cluster.
+
+## Troubleshoot Node Removal
+
+This section includes information about troubleshooting issues with node removal in Rook Ceph clusters.
+
+### Rook Ceph v1.0.4 is Unhealthy with Mon Pods Not Rescheduled
+
+#### Symptom
+
+When you run `kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status` for a Rook Ceph v1.0.4 cluster, you see that Ceph is in an unhealthy state where failed mons are not rescheduled. 
+
+For example:
+
+```
+health: HEALTH_WARN
+        1/3 mons down, quorum a,c
+```
+
```

Comment:
I think we need to add here the mon pod in Pending state as well;

![mon-pod-peding](https://user-images.githubusercontent.com/7708031/214511317-2da67544-65b8-4b0a-a4a5-14057f5b0271.png)

Reply:
Will do! Question: Will the value of the `app` field vary depending on the name of the application? `rook-ceph-mon` is just an example app that you've creating for testing, correct?

(I'll update the command to make it clear that this is a placeholder value if so)

Reply:
Done!